paparazzi-preview-test-junit/src/main/java/app/cash/paparazzi/preview/Snapshot.kt

```diff
@@ -0,0 +1,36 @@
+// Copyright Square, Inc.
+package app.cash.paparazzi.preview
+
+import androidx.compose.runtime.Composable
+import app.cash.paparazzi.Paparazzi
+import app.cash.paparazzi.annotations.PaparazziPreviewData
+import com.google.testing.junit.testparameterinjector.TestParameter.TestParameterValuesProvider
+
+/**
+ * Take a snapshot of the given [previewData].
+ */
+public fun Paparazzi.snapshot(
+  previewData: PaparazziPreviewData,
+  name: String? = null,
+  wrapper: @Composable (@Composable () -> Unit) -> Unit = { it() }
+) {
+  when (previewData) {
+    is PaparazziPreviewData.Default -> snapshotDefault(previewData, name, wrapper)
+    is PaparazziPreviewData.Empty -> Unit
+    is PaparazziPreviewData.Error -> throw Exception(previewData.message)
```

Comment:
maybe just use kotlin helper
```suggestion
    is PaparazziPreviewData.Error -> error(previewData.message)
```